api.php

```diff
@@ -126,8 +163,41 @@
 $app->post('/user/new', function (Request $request, Response $response, array $args) {
 	// get request body
 	$body = json_decode($request->getBody());
+	
+	/* Check phone number
+	Accepted patterns for phone-number
+	###-###-####
+	(###)###-####
+	(###) ###-####
+	##########
+	### ### ####
+	*/
+
+	//regex pattern set to a string
+	$numberRegEx = '\(?(\d{3})[\)\s-]*(\d{3})[\s\-]?(\d{4})';
+	//check user input against string pattern
+	if (!preg_match($numberRegEx, $body->phone)) {
+		//set response message for invalid format
+		$response->getBody()->write(
+			json_encode(
+				['message' => 'invalid format for phone number']
+			)
+		);
+		//return the error with an invalid request status
+		return $response
+			->withHeader('Content-Type', 'application/json')
+			->withStatus(400);
+	}
+	
+	//return phone output (###) ###-####
+	//regx pattern set as a string
+	$userPhoneRegex = '\(?(\d{3})[\)\s-]*(\d{3})[\s\-]?(\d{4})';
+	
+	//output of function set to userPhone 
```

Comment:
In what format?

Reply:
commit 9968cc6 
Added to comment -> (###) ###-####